content/src/service/garbage-collection/GarbageCollectionManager.ts

```diff
@@ -47,24 +47,17 @@ export class GarbageCollectionManager {
   async performSweep() {
     const newTimeOfCollection: number = Date.now()
     this.sweeping = true
+    const { end: endTimer } = this.components.metrics.startTimer('dcl_content_garbage_collection_time')
     try {
-      await this.components.repository.tx(
-        async (transaction) => {
-          const { end: endTimer } = this.components.metrics.startTimer('dcl_content_garbage_collection_time')
+      const hashes = await findContentHashesNotBeingUsedAnymore(this.components, this.lastTimeOfCollection)
 
-          const hashes = await transaction.content.findContentHashesNotBeingUsedAnymore(this.lastTimeOfCollection)
+      this.components.metrics.increment('dcl_content_garbage_collection_items_total', {}, hashes.length)
 
-          this.components.metrics.increment('dcl_content_garbage_collection_items_total', {}, hashes.length)
+      this.LOGGER.debug(`Hashes to delete are: (${hashes.join(',')})`)
```

Comment:
is debug logging working okay now? If not, would that be too much noise?

Reply:
It logs a considerable amount of text but every long period of time i think. I can change it if you think it's better